packages/2018-disaster-resilience/src/components/EarthquakeDamageEstimatesForBuildingsInTillamookCounty/EarthquakeDamageEstimatesForBuildingsInTillamookCountyVisualization.js

```diff
@@ -76,14 +78,37 @@ const EarthquakeDamageEstimatesForBuildingsInTillamookCountyVisualization = ({
               civicMapStyle={mapStyles[dataType].map}
             >
               <ScreenGridMap
-                data={data.damageEstimates.value}
+                data={data.damageEstimates.value.filter(
+                  feature => feature.properties[mapStyles[dataType].field]
+                )}
```

Comment:
In the future, it's nice when values like this are abstracted to a well named variable so that it's clear how the data has been transformed.